src/05-derivatives/02-common-data-types.md

```diff
@@ -0,0 +1,59 @@
+# Common data types
+
+## Preprocessed or cleaned data
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        <datatype>/
+            <source_keywords>[_space-<space>][_desc-<label>]_<suffix>.<ext>
+```
+
+Preprocessing in this context means transformations of data that do not change
+the data type of the input (as expressed by its BIDS `suffix`).
+For instance, a change in the number of dimensions is likely to disrupt the propagation
+of the input's `suffix` and generally, the outcomes of such transformation
+cannot be considered _preprocessed_ or _cleaned data_.
+Examples:
+
+ -  Motion-corrected, temporally denoised, and transformed to MNI space BOLD series
+ -  Inhomogeneity corrected and skull stripped T1w files
+ -  Motion-corrected DWI files
+ -  Time-domain filtered EEG data
+ -  MaxFilter (for example, SSS) cleaned MEG data
+
+The `space` keyword is recomended to distinguish files with different underlying
```

Comment:
Please use the correct terminology: `space` is not a "keyword" but it is an "entity" as defined per https://bids-specification.readthedocs.io/en/common-derivatives/02-common-principles.html#file-name-structure.

AFAIK the only terms in the specification that are known as "keywords" are MUST, SHOULD, etc.